Kalela/BrightEventsReactApp
# Ft-ui-ux-design-156802041

### What does this PR do?
Sets up new UI for the react app.
### How should this be manually tested?
- Clone the repo locally.
- cd into the project.
- Make sure you have npm installed. Check installation [here](https://www.codecademy.com/articles/react-setup-i)
- Run ```npm install -g``` to install all the packages documented in package.json.
- Run ```npm start``` in your terminal to get the server up.
- Visit the site on ```localhost:3000``` and view the new UI.

### Screenshot
![image](https://user-images.githubusercontent.com/33759757/38931003-74bd5fea-431a-11e8-90a4-120f97ef4a8d.png)
`The new theme with less color mixing and a minimalist design`
### What are the relevant pivotal tracker stories?
Create better UI
ID: #156802041
